modules/google_service_account/inputs.tf

```diff
@@ -59,3 +59,57 @@ variable "project" {
   description = "The ID of the project that the service account will be created in. Defaults to the provider project configuration."
   type        = string
 }
+
+variable "in_project_condition_title" {
```

Comment:
So we’re supporting only one condition per service account? Or what’s the limitation?

Reply:
Yes we are only supporting one. The docs (https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/google_folder_iam#google_folder_iam_member) don't really describe how you can have more

Reply:
Blocks like that can be defined as many times as you want, you can see from the example directly below wit it's `audit_log_config`. Your code basically already supports this anyway.

```
resource "google_folder_iam_audit_config" "folder" {
  folder  = "folders/1234567"
  service = "allServices"
  audit_log_config {
    log_type = "ADMIN_READ"
  }
  audit_log_config {
    log_type = "DATA_READ"
    exempted_members = [
      "user:joebloggs@hashicorp.com",
    ]
  }
}
 ```